allocations/allocations_api.py

```diff
@@ -0,0 +1,36 @@
+import requests
+import json
+from django.conf import settings
+import logging
+from keystoneauth1.identity import v3
+from keystoneauth1 import session
+
+logger = logging.getLogger('default')
+
+class BALANCE_SERVICE_API:
+
+    def call( self, project_code ):
+        # get token
+        auth = v3.Password(auth_url=settings.OPENSTACK_KEYSTONE_URL,
+                           username=settings.OPENSTACK_SERVICE_USERNAME,
+                           password=settings.OPENSTACK_SERVICE_PASSWORD,
+                           project_id=settings.OPENSTACK_SERVICE_PROJECT_ID,
+                           user_domain_name='default',
+                           user_domain_id='default')
+        sess = session.Session(auth=auth)
+        auth_token = sess.get_token()
```

Comment:
I think there is even a `get_headers` or something, which might be the more future-proof solution. It would return a headers dict, which you can then add your content-type stuff to.